addons/web/static/src/legacy/js/views/basic/basic_renderer.js

```diff
@@ -737,6 +737,7 @@ var BasicRenderer = AbstractRenderer.extend(WidgetAdapterMixin, {
             // This is needed as modifiers are set after first render
             hasReadonlyModifier: modifiers.readonly,
             mode: modifiers.readonly ? 'readonly' : mode,
+            actualMode: mode,
```

Comment:
I'm really not in favor of adding an option.
If that were to prove indispensable, i'd rather have a `isViewReadonly` boolean or something of the sort....

Reply:
Hello @msh-odoo 

Thanks for the dev :)

I agree with @kebeclibre that the ``actualMode`` is not meaningful out of the context of this PR. Could you please adapt the branch by using an attribute ``isViewReadonly`` which is a boolean ? 

Ping me once it's ready, and I will proceed to the r+ thing. 

Thanks :)